Color Picker/ColorPickerScreen.swift

```diff
@@ -155,222 +282,83 @@ struct ColorPickerScreen: View {
-				.buttonStyle(.borderless)
-				.contentShape(.rectangle)
-				.keyboardShortcut("h", modifiers: [.shift, .command])
-		}
-	}
-
-	private var hslColorView: some View {
-		HStack {
-			NativeTextField(
-				text: $hslColor,
-				placeholder: "HSL",
-				font: .monospacedSystemFont(ofSize: textFieldFontSize, weight: .regular),
-				isFocused: $isTextFieldFocusedHSL
-			)
-				.controlSize(.large)
-				.onChange(of: hslColor) {
-					if
-						isTextFieldFocusedHSL,
-						!isPreventingUpdate,
-						let newColor = NSColor(cssHSLString: $0.trimmingCharacters(in: .whitespaces))
-					{
-						colorPanel.color = newColor
-					}
-
-					if !isPreventingUpdate {
-						updateColorsFromPanel(excludeHSL: true, preventUpdate: true)
-					}
-				}
-			Button("Copy HSL", systemImage: "doc.on.doc.fill") {
-				hslColor.copyToPasteboard()
-			}
-				.labelStyle(.iconOnly)
-				.symbolRenderingMode(.hierarchical)
-				.buttonStyle(.borderless)
-				.contentShape(.rectangle)
-				.keyboardShortcut("s", modifiers: [.shift, .command])
-		}
-	}
-
-	private var rgbColorView: some View {
-		HStack {
-			NativeTextField(
-				text: $rgbColor,
-				placeholder: "RGB",
-				font: .monospacedSystemFont(ofSize: textFieldFontSize, weight: .regular),
-				isFocused: $isTextFieldFocusedRGB
-			)
-				.controlSize(.large)
-				.onChange(of: rgbColor) {
-					if
-						isTextFieldFocusedRGB,
-						!isPreventingUpdate,
-						let newColor = NSColor(cssRGBString: $0.trimmingCharacters(in: .whitespaces))
-					{
-						colorPanel.color = newColor
-					}
-
-					if !isPreventingUpdate {
-						updateColorsFromPanel(excludeRGB: true, preventUpdate: true)
-					}
-				}
-			Button("Copy RGB", systemImage: "doc.on.doc.fill") {
-				rgbColor.copyToPasteboard()
-			}
-				.labelStyle(.iconOnly)
-				.symbolRenderingMode(.hierarchical)
-				.buttonStyle(.borderless)
-				.contentShape(.rectangle)
-				.keyboardShortcut("r", modifiers: [.shift, .command])
-		}
-	}
-
-	private var lchColorView: some View {
-		HStack {
-			NativeTextField(
-				text: $lchColor,
-				placeholder: "LCH",
-				font: .monospacedSystemFont(ofSize: textFieldFontSize, weight: .regular),
-				isFocused: $isTextFieldFocusedLCH
-			)
-				.controlSize(.large)
-				.onChange(of: lchColor) {
-					if
-						isTextFieldFocusedLCH,
-						!isPreventingUpdate,
-						let newColor = NSColor(cssLCHString: $0.trimmingCharacters(in: .whitespaces))
-					{
-						colorPanel.color = newColor
-					}
-
-					if !isPreventingUpdate {
-						updateColorsFromPanel(excludeLCH: true, preventUpdate: true)
-					}
-				}
-			Button("Copy LCH", systemImage: "doc.on.doc.fill") {
-				lchColor.copyToPasteboard()
-			}
-				.labelStyle(.iconOnly)
-				.symbolRenderingMode(.hierarchical)
-				.buttonStyle(.borderless)
-				.contentShape(.rectangle)
-				.keyboardShortcut("l", modifiers: [.shift, .command])
-		}
-	}
-
 	var body: some View {
 		VStack {
 			BarView()
 			if shownColorFormats.contains(.hex) {
```

Comment:
You can just loop over the enum instead of manually defining each view. That was the point of extracting it into a reusable view.